go/vm_power_cycle_test.go

```diff
@@ -0,0 +1,160 @@
+package testGoSDK
```

Comment:
Just a note that these tests may be unstable due to xapi's behaviour. Let's try running them for now, but we may end up disabling them like the Java tests here https://github.com/xenserver/xenserver-samples/pull/17